webserver/views/api/v1/core.py

```diff
@@ -109,3 +111,20 @@ def _validate_offset(offset):
     else:
         offset = 0
     return offset
+
+@bp_core.route("/<user_api_key>/get_pending_jobs", methods=["GET"])
+def get_pending_jobs(user_api_key):
+    """ Return the pending jobs for a user.
+
+    Retrieve the Job Ids for the pending jobs for the user whose
+    user id is given as an argument and return that.
+    """
+    final_result = {}
+    try:
+        pending_jobs = db.dataset_eval.get_user_pending_jobs(user_api_key)
+        for i in range(len(pending_jobs)):
```

Comment:
However, if you wanted to do this for some other reason in the future, a good tool here is `enumerate`:
```python
>>> for i, item in enumerate(["a", "list", "here"], 1):
...     print i, item
... 
1 a
2 list
3 here
```
the second argument is the number to start counting from. by default it starts from 0